blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -111,11 +108,10 @@ public void startServer() {
         Channel serverChannel = server.bind(new InetSocketAddress(httpIngestHost, httpIngestPort));
         allOpenChannels.add(serverChannel);
 
-        log.info("Starting tracker service");
-        tracker = new Tracker();
+        log.info("Registering tracker service");
+        Tracker.getInstance();
```

Comment:
Note that if I don't have this call here, the track() laters still work (all tests past and all code functionalities within blueflood will work as intended), but we will not be able to see the MBean on service startup (because it hasn't been registered yet) and will not be able to make any of the calls to the MBean (such as addTenant, addMetricName, setIsTrackingDelayedMetrics) until at least one .track() call is made, at which time the registration is called from the private constructor.